pkg/reconciler/tenancy/clusterworkspacetype/clusterworkspacetype_controller_reconcile.go

```diff
@@ -69,13 +68,7 @@ func (c *controller) updateVirtualWorkspaceURLs(cwt *tenancyv1alpha1.ClusterWork
 		u, err := url.Parse(clusterWorkspaceShard.Spec.ExternalURL)
 		if err != nil {
 			// Should never happen
-			klog.Errorf(
-				"Error parsing ClusterWorkspaceShard %s|%s spec.externalURL %q: %v",
-				logicalcluster.From(clusterWorkspaceShard),
-				clusterWorkspaceShard.Name,
-				clusterWorkspaceShard.Spec.ExternalURL,
-			)
-
+			logger.Error(err, "error parsing clusterWorkspaceShard.spec.externalURL")
```

Comment:
Not sure if we need to add back including the externalURL as a log value?

Reply:
I don't know that you need the value *and* the error, but sure - can't hurt